content/agent/kubernetes/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Kubernetes
+kind: documentation
+aliases:
+    - /guides/basic_agent_usage/kubernetes
+    - /agent/basic_agent_usage/kubernetes
+further_reading:
+- link: "agent/kubernetes/daemonset_setup"
+  tag: "documentation"
+  text: "Kubernetes DaemonSet Setup"
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/metrics"
+  tag: "documentation"
+  text: "Kubernetes Metrics"
+- link: "agent/kubernetes/legacy"
+  tag: "documentation"
+  text: "Kubernetes Legacy Versions"
+
+---
+
+**Note**: Agent version 6.0 and above only support versions of Kubernetes higher than 1.7.6. For prior versions of Kubernetes, consult the [Legacy Kubernetes versions section][1].
+
+## Installation
+
+To gather metrics, traces, and logs from your Kubernetes clusters, there are two options:
+
+* [Container installation][2] (**recommended**) -  The Agent runs inside a Pod. This implementation is sufficient for the majority of use cases, but note that it does not grant visibility into components of the system that exist outside Kubernetes. This method also does not monitor the starting phase of your Kubernetes cluster.
+* [Host installation][3] (optional) - Installing the Agent on the host provides additional visibility into your ecosystem, independent of Kubernetes.
+
+## RBAC
+
+In the context of using the Kubernetes integration, and when deploying Agents in a Kubernetes cluster, a set of rights are required for the Agent to integrate seamlessly.
+
+You will need to allow the Agent to perform a few actions:
```

Comment:
r / `You will need to allow the Agent to perform a few actions:` / `You must allow the Agent to perform a few actions`: